src/App.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from "react";
+import "./App.css";
```

Comment:
The rule of thumb is to keep the import order like this:

● Built-in
● External
● Internal